### PR TITLE
fixing typos in message concept doc

### PR DESCRIPTION
**Pull request by fhs-improving**

### CODE REVIEW

File: doc/src/hugo/content/concepts/message.md [hunk 1]
```diff
@@ -8,12 +8,12 @@ an implementation that is a message-driven system per the
 [Reactive Manifesto](https://reactivemanifesto.org). Messages in RIDDL are 
 a special case of an [aggregate type]({{< relref "type.md#aggregation" >}}) 
 and the _lingua franca_ of many RIDDL definitions. They define the API for:
-* [`adaptor`s]({{< relref "adaptor.md" >}})
```
Comment (fhs-improving) [inline doc/src/hugo/content/concepts/message.md:11]: The s can be inside or outside the backticks but it should be consistent across all items; move it inside here or outside below on repositories

Comment (reid-spencer) [reply doc/src/hugo/content/concepts/message.md:11]: It was consistently outside the backticks which is where it should be. we only want to use backticks around the actual RIDDL language keyword. Please revert.

**Review by reid-spencer — CHANGES_REQUESTED** [review]
Mostly okay. Please revert the placement of the s to be consistently outside the backticks.

**Review by reid-spencer — APPROVED** [review]
LGTM

**Pull request merged**